README.md

```diff
@@ -50,6 +50,7 @@ jobs:
 - **create_annotated_tag** _(optional)_ - Boolean to create an annotated rather than a lightweight one (default: `false`).
 - **tag_prefix** _(optional)_ - A prefix to the tag name (default: `v`).
 - **append_to_pre_release_tag** _(optional)_ - A suffix to the pre-release tag name (default: `<branch>`).
+- **initial_prerelease_bump** _(optional)_ - A default prerelease bump to apply if no prerelease information is present on the latest tag. Defaults to prepatch.
```

Comment:
Let move this up by the default_prerelease_bump

Reply:
You should also list all the possible values as in default_bump and default_prerelease_bump

Reply:
Done